src/components/BoxForm/DropZone/DropZone.jsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import React, { useMemo, useState, useEffect } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { CloseIcon } from '@chakra-ui/icons';
+import PropTypes from 'prop-types';
+import './DropZone.css';
+
+function DropZone({ setFiles }) {
```

Comment:
Change component to arrow function to follow ES6 conventions and so that we can keep things consistent throughout